app/controllers/users.js

```diff
@@ -82,3 +85,133 @@ exports.signin = (request, response, next) => {
       next(err);
     });
 };
+
+exports.listUsers = (request, response, next) => {
+  const userLogged = request.user;
+  if (userLogged) {
+    const limit = request.query.limit ? parseInt(request.query.limit) : 10;
+    const offset = request.query.offset ? parseInt(request.query.offset) : 0;
+
+    userService.getAllUsersPaginate(limit, offset).then(users => {
+      if (users != null) {
```

Comment:
Try with an early return here, idk if the whole line will be greather than the permitted by the linter